css/about.css

```diff
@@ -0,0 +1,70 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400;500;600;700&family=Poppins:wght@100;200;300;400;500;600;700&family=Roboto:wght@100;300;400;500;700&display=swap");
+
+.about-section {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr;
+}
+
+.about-section-container {
+  background: #1c1a19;
+}
+
+.about-heading {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  margin-top: 120px;
+  display: flex;
+  justify-content: center;
+}
+
+.about-para {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 30px;
+  color: #f4f5f7;
+  margin: 24px;
+}
+
+.about-btn {
+  flex-direction: row;
+  padding: 12px;
+  gap: 10px;
+  width: 156px;
+  height: 48px;
+  background: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  margin-left: 24px;
+  border: transparent;
+}
```

Comment:
- [ ] Please avoid duplication I noticed you create different classes for buttons please use `DRY` principles. The `DRY(Don't Repeat Yourself)` principle states that only one copy of crucial information should exist. Cascading style sheets control the appearance of `HTML` documents. Having separate `HTML` and `CSS` files makes it easy and efficient to have several different `HTML` documents with the same general appearance. I also recommend this [article](https://medium.com/circuits-technical-talks/dry-css-9cbe3215af3d) for understand of DRY concept :slightly_smiling_face: